urllib3/response.py

```diff
@@ -200,6 +202,17 @@ def _init_length(self, request_method):
         Logic extracted from httplib.py in the standard Python library.
         """
         length = self.headers.get('content-length')
+
+        if length is not None and self.chunked:
+            # This Response will fail with an IncompleteRead if it can't be
+            # received as chunked. This works as a fallback to at least attempt
```

Comment:
Could probably clarify "This works as a fallback to at least attempt to read" as "This method falls back to attempt reading..."